ppr-ui/src/components/mhrTransfers/TransferDetails.vue

```diff
@@ -180,7 +180,7 @@ export default defineComponent({
       declaredValue: getMhrTransferDeclaredValue.value?.toString(),
       consideration: getMhrTransferConsideration.value,
       transferDate: getMhrTransferDate.value,
-      isOwnLand: getMhrTransferOwnLand.value,
+      isOwnLand: getMhrTransferOwnLand.value === null ? false : getMhrTransferOwnLand.value,
```

Comment:
isOwnLand: getMhrTransferOwnLand.value || false

Reply:
ok

Reply:
Missed this change.

Reply:
not sure how that happened... Anyways... I have fixed the issue.